test/cmd_line/vsplit.test.ts

```diff
@@ -0,0 +1,65 @@
+import { ModeHandler } from '../../src/mode/modeHandler';
+import { setupWorkspace, cleanUpWorkspace, assertEqualLines, assertEqual } from './../testUtils';
+import { runCmdLine } from '../../src/cmd_line/main';
+import * as vscode from 'vscode';
+import { join } from 'path';
+import * as assert from 'assert';
+import { getAndUpdateModeHandler } from '../../extension';
+
+async function WaitForEditorOpen(): Promise<void> {
```

Comment:
Consider making this function a bit more generic. Maybe accept an "expectedEditorOpen" param.